tsconfig.compodoc.json

```diff
@@ -148,14 +191,36 @@
       "@spartacus/checkout/scheduled-replenishment/root": [
         "feature-libs/checkout/scheduled-replenishment/root/public_api"
       ],
-      "@spartacus/order/assets": ["feature-libs/order/assets/public_api"],
+      "@spartacus/customer-ticketing/assets": [
+        "feature-libs/customer-ticketing/assets/public_api"
+      ],
+      "@spartacus/customer-ticketing/core": [
+        "feature-libs/customer-ticketing/core/public_api"
+      ],
+      "@spartacus/customer-ticketing": [
```

Comment:
same case here

missing /occ

Reply:
Will stop flagging, but

it needs to be updated for these files. Once you get prettier with the same setup, please re-save these files to not include all these extra break lines

```
tsconfig.json,
tsconfig.compodoc.json,
projects/storefrontapp/tsconfig.server.prod.json,
projects/storefrontapp/tsconfig.server.json,
projects/storefrontapp/tsconfig.app.prod.json
```